editor.html

```diff
@@ -358,6 +358,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
       'blocks': false,
       'snippets': true,
       'experimental': false,
+      'fs': true,
```

Comment:
We are not using feature flags for the filesystem, so we can always include the information in the help.html file without processing query parameters.

Reply:
Ack. I've removed the flag references in https://github.com/bbcmicrobit/PythonEditor/pull/114/commits/95c3d9638287a26446796d500ee62640f0585e3b